tests/tests.py

```diff
@@ -67,6 +66,83 @@ def test_piranha_match_only():
             assert rule 
             assert _is_readable(str(match))
 
+
+def test_insert_field_add_import():
+    add_field_declaration = Rule (
+        name= "add_field_declaration",
+        query= 
+        """(
+            (class_declaration name: (_)@class_name 
+                body : (class_body ((_)*) @class_members)  @class_body
+            ) @class_declaration
+            (#eq? @class_name "FooBar")
+        )
+        """,
+        replace_node="class_body",
+        replace="""{
+  private List<String> names;\n @class_members
+}""",
```

Comment:
I guess one interesting caveat of the python API is that multi-line strings force us to have some somewhat awkward indentation here in order to get the right indentation in the final file. Finally one point in favor of the otherwise annoying EP varargs `lines` approach (see [here](https://github.com/uber/NullAway/blob/master/nullaway/src/test/java/com/uber/nullaway/NullAwayCoreTests.java#L74) for example)